src/main/java/org/geotools/data/neo4j/Neo4jSpatialDataStore.java

```diff
@@ -167,9 +167,8 @@ protected ContentFeatureSource createFeatureSource(ContentEntry contentEntry) th
         Neo4jSpatialFeatureSource source = new Neo4jSpatialFeatureSource(contentEntry, database, layer, buildFeatureType(contentEntry.getTypeName()), records, extraPropertyNames);
         if (layer instanceof EditableLayer) {
             return new Neo4jSpatialFeatureStore(contentEntry, database, (EditableLayer) layer, source);
-        } else {
-            return source;
         }
+		return source;
```

Comment:
Please do not change the indent to tab in this PR (we need to add an additional refomrat PR for this)

Reply:
I created a PR for this: https://github.com/neo4j-contrib/spatial/pull/396